### PR TITLE
Update OpenStack stub editing instructions

### PR DESCRIPTION
**Pull request by Amit-PivotalLabs**
also minor touch-up on AWS stub editing instructions

### CODE REVIEW

**Comment by animatedmax** [comment]
Thank you, Amit-PivotalLabs

File: openstack/cf-stub.html.md.erb [hunk 1]
```diff
@@ -21,223 +21,229 @@ information about your environment.
...
-    <code>bosh status</code> to view the BOSH Director UUID.</td>
+    <code>bosh status --uuid</code> to view the BOSH Director UUID.
+    </td>
+  </tr>
+  <tr>
+    <td><pre><code>
+meta:
+  environment: ENVIRONMENT
+    </code></pre></td>
+    <td>Replace <code>ENVIRONMENT</code> with an arbitrary name describing your environment, e.g. <code>aws-prod</code>.
```
Comment (voelzmo) [inline openstack/cf-stub.html.md.erb]: typo: 'aws' -> 'openstack'

File: openstack/cf-stub.html.md.erb [hunk 2]
```diff
@@ -21,223 +21,229 @@ information about your environment.
...
+          -----BEGIN CERTIFICATE-----
+          SSL_CERTIFICATE_SIGNED_BY_PRIVATE_KEY
+          -----END CERTIFICATE-----
     </code></pre></td>
-	<td>Replace <code>uaaadmin</code> with the admin user name used to connect
-	to the UAA database.
-	<br /><br />
-	Replace <code>admin_password</code> with the UAA database admin password.
-	</td>
+    <td>Replace <code>RSA_PRIVATE_KEY</code> and <code>SSL_CERTIFICATE_SIGNED_BY_PRIVATE_KEY</code> with the PEM-encoded pirvate key and certificate associated with the system domain and apps domains you've configured to terminate at the floating IP associated with the <code>ha_proxy</code> job.
```
Comment (voelzmo) [inline openstack/cf-stub.html.md.erb]: typo: 'pirvate' -> 'private'

Comment (animatedmax) [inline openstack/cf-stub.html.md.erb]: Fixed, thank you voelzmo

File: openstack/cf-stub.html.md.erb (hunk 1 above)
Comment (animatedmax) [inline openstack/cf-stub.html.md.erb]: Fixed, thank you voelzmo

**Pull request merged**